generic3g/tests/Test_MeanTransform.pf

```diff
@@ -40,9 +40,9 @@ contains
 
       ! counter 0 at one point
       call FieldSet(acc%accumulation_field, COUNTER*MEAN, _RC)
-      call assign_fptr(acc%counter_field, fptr, _RC)
-      fptr(n) = 0
-      mask = fptr /= 0
+      call assign_fptr(acc%counter_field, ifptr, _RC)
+      ifptr(n) = 0
+      mask = ifptr /= 0
```

Comment:
Does this mean the PR actually found a bug in an existing test?  (Just want to be sure I understood.)

Reply:
Yes.